test/opsem/bswap-ntohs-inline-asm_sat.ll

```diff
@@ -0,0 +1,81 @@
+; RUN: %seabmc "%s" 2>&1 | %oc %s
```

Comment:
As far as i can tell... the `ll` file is an output for the `c` source. We only need one usually. In this case to test that `ror` --> bswap --> ir, I would assume we necessarily need the `c` file but not the `ll` file.

Reply:
Arie suggests us to remain `ll` files. I made new updates and removed `c` files.